src/lib/rmt/wizard_scc_page.rb

```diff
@@ -26,6 +26,8 @@ module RMT; end
 class RMT::WizardSCCPage < Yast::Client
   include ::UI::EventDispatcher
 
+  YAST_RMT_USER_AGENT = 'yast2-rmt'.freeze
```

Comment:
What about adding some version number to the agent string? Standard web browsers do that as well.

Non-working pseudocode:
```ruby
YAST_RMT_USER_AGENT = "yast2-rmt #{RMT::VERSION}".freeze
```

In theory this could allow to handle some corner cases or bugs in SMT on the server size. Or if the SCC changes the API it could fallback to old API for old versions...

Ideally this should not be needed but it would be nice to have this possibility as the last resort...

Reply:
I created the constant module `RMT::VERSION`, added it to the user agent, and created a unit test to ensure it always matches the package version. Please check commit 89e919c6ba0cb98f7882ce158f603a3e495c23f0.